controllers/animal_c.js

```diff
@@ -11,14 +11,14 @@ animalController.getAllAnimals = async function (req, res) {
   // #swagger.responses[200] = {description: "Success"}
   // #swagger.responses[500] = {description: "Internal Server Error"}
   try {
-    const result = await db.getDatabase().db('WildlifeAPI').collection('Animals').find();
-    result.toArray().then((animals) => {
```

Comment:
The use of ".then" is an alternative way to do asynchronous JS. It effectively means "await whatever is to the left, then call the following function on its return value". The change is unnecessary because it is simply a more verbose way of doing the same thing.

Reply:
I left it like this because throughout the project, await has been used in almost all asynchronous calls instead of then.